### DISCUSSION TITLE
find-replace plugin

### DISCUSSION THREAD

**Discussion by fEyebrow** [ueberdosis/tiptap#3380] [Community Extensions]
I came across some problems when creating find-replace plugin:
https://github.com/fEyebrow/editor/blob/131c857026d85c7a5a5b5a9624fbbd9adf624db5/src/extensions/findReplace.ts#L45

<img width="451" alt="image" src="https://user-images.githubusercontent.com/18180417/199381640-59287d16-41ee-45ad-a4a8-f2fe05a54f45.png">

**Comment by fEyebrow** [comment]
I solved the problem

**Reply by lzxlzxlzxl** [reply]
How do you implement find and replace plugins？

**Comment by sereneinserenade** [comment]
I've made a search and replace extension for this purpose. Repo: https://github.com/sereneinserenade/tiptap-search-and-replace
